src/main/java/org/openmrs/addonindex/service/ElasticSearchIndex.java

```diff
@@ -98,8 +105,25 @@ public void index(AddOnInfoAndVersions infoAndVersions) throws IOException {
 	}
 	
 	@Override
-	public Collection<AddOnInfoSummary> search(AddOnType type, String query, String tag) throws IOException {
+	public Collection<AddOnInfoSummary> search(AddOnType type, String query, String tag, String uid, String moduleId,
+											   String name, String exclude, AddOnStatus status) throws IOException {
 		BoolQueryBuilder boolQB = QueryBuilders.boolQuery();
+		if (name != null) {
+			//Exact match on name
+			boolQB.filter(QueryBuilders.matchQuery("name.raw", name));
```

Comment:
this should be case-insensitive.
probably all of them should be. Are they?

Reply:
Yes, all are case-insensitive. Keywords are by default case-sensitive. To make this field case-insensitive is why I added a [normalizer](https://www.elastic.co/guide/en/elasticsearch/reference/5.2/normalizer.html) to it(this feature is only available starting v5.2 of elasticsearch. So, I guess we should update the ReadME to include this)

Reply:
Please update the README. Is that about also sourcing the -setting.json file, or do we also need to update the deployment scripts to use a later version of elasticsearch?